### ISSUE TITLE
[Feature] How can I calculate the loss on evaluation dataset?

### ISSUE THREAD

**Issue by liming-ai** [open-mmlab/mmpose#2948]
### What is the feature?

In some situations, I want to visualize the loss on the validation dataset, how could I achieve this feature with pre-trained models? Let's say DKER for example.

**Comment by Tau-J** [comment]
Hi @liming-ai , thanks for using MMPose. You can modify the `predict()`  and add the code of loss calculation.

**Comment by liming-ai** [comment]
> Hi @liming-ai , thanks for using MMPose. You can modify the `predict()` and add the code of loss calculation.

Thanks a lot.

**Comment by ElviraLL** [comment]
Hi @Tau-J Thanks for the information. Can you also give some suggestions on how to pass the calculated loss back to logger_hook? Thanks!
